### ISSUE TITLE
docker image not found

### ISSUE THREAD

**Issue by behrad** [ottemo/foundation-stable#2]
is this the ottemo-ng backend? I couldn't docker pull it :(

**Comment by vastbinderj** [comment]
yes - @maazghani is working on the docker documentation.  For now, you can download the binary from the releases for your platform.  I can include the Docker file in the releases starting this coming weekend, on the 30th.

You can hit us up in our public hipchat conference room if you have questions.  This is a very early release and the documentation isn't nearly ready.

**Comment by behrad** [comment]
eager to check it out @vastbinderj :)
Is there a possiblity to extend/customize the source in that docker image?

**Comment by vastbinderj** [comment]
@behrad at this time, we are only shipping the binaries.  We are working on a partner program to get the source code into hands of developers for Foundation.  You do have full access to the source code for our store front-end and the store administration tool.
